src/applications/ds-playground/pages/V1V3Page.jsx

```diff
@@ -492,6 +493,34 @@ export default function V1V3Page() {
             </div>
           </div>
         </div>
+
+        {/* Progress bar - segmented */}
+        <div className="vads-l-row">
+          <h3>Progress bar - segmented</h3>
+          <div className="vads-l-col--12 vads-u-align-items--center vads-u-border-bottom--1px vads-u-border-color--primary medium-screen:vads-u-display--flex">
+            <div className="vads-l-col--12 small-screen:vads-l-col--6 vads-u-margin--1">
+              <VaSegmentedProgressBar
+                current={2}
+                heading-text="V1 VA Benefits"
+                total={5}
+                uswds
```

Comment:
Should the V1 progress bar not have the uswds flag?

Reply:
right, my bad